book/src/livenessprobe.md

```diff
@@ -1,22 +1,20 @@
 # CSI `livenessprobe`
 
-## Status
+## Status and Releases
 
-GA/Stable
+**Git Repository:** https://github.com/kubernetes-csi/livenessprobe
 
-## Details
-
-Git Repository: https://github.com/kubernetes-csi/livenessprobe
-
-The CSI `livenessprobe` is a sidecar container that monitors the health of the CSI driver and reports it to Kubernetes via the [Liveness Probe mechanism](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/). This enables Kubernetes to automatically detect issues with the driver and restart the pod to try and fix the issue.
-
-## Stable Releases
+**Status:** GA/Stable
 
 Latests stable release | Branch | Compatible with CSI Version | Container Image | Min k8s Version | Max k8s version
 --|--|--|--|--|--
 [livenessprobe v1.0.2](https://github.com/kubernetes-csi/livenessprobe/releases/tag/v1.0.2) | [release-1.0](https://github.com/kubernetes-csi/livenessprobe/tree/release-1.0) | [v1.0.0](https://github.com/container-storage-interface/spec/releases/tag/v1.0.0) | quay.io/k8scsi/livenessprobe:v1.0.2 | v1.13 | -
 Unsupported. | No 0.x branch. | [v0.3.0](https://github.com/container-storage-interface/spec/releases/tag/v0.3.0) | quay.io/k8scsi/livenessprobe:v0.4.1 | v1.10 | -
 
+## Description
+
+The CSI `livenessprobe` is a sidecar container that monitors the health of the CSI driver and reports it to Kubernetes via the [Liveness Probe mechanism](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/). This enables Kubernetes to automatically detect issues with the driver and restart the pod to try and fix the issue.
```

Comment:
restart the pod, or just the driver container?

Reply:
I forget. @sbezverk can you clarify?

Reply:
@saad-ali it should say restart the driver container.